app/views/content/life-as-a-teacher/explore-subjects/physical-education/_article.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach PE</h2>
+      <p><p>Becoming a PE teacher means more than teaching sports. It’s about inspiring pupils to embrace a healthy and active lifestyle, and helping them unlock their potential.</p>
+<p>Your teaching will encourage them to discover the benefits of movement, both for their physical fitness and overall wellbeing.</p>
+   <%= render 'content/shared/teaching/benefits' %>
+   <%= render 'content/shared/teaching/why_teach' %>
+  </section>
+
+<section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+   <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching PE opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from sports coaching and personal training, to sports therapy or nutrition.</p>
+      <p>Your pupils will learn more from you than you might think. As their teacher, you’ll teach them the skills to develop perseverance and face challenges with determination.</p>    </div>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-physical-education-programmes-of-study">national curriculum for PE</a>.</p>
```

Comment:
Don't we normally link directly to the curriculum page?
eg
https://www.gov.uk/government/publications/national-curriculum-in-england-physical-education-programmes-of-study/national-curriculum-in-england-physical-education-programmes-of-study